x/distribution/autocli.go

```diff
@@ -135,12 +135,16 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 					},
 				},
 				{
-					RpcMethod: "UpdateParams",
-					Skip:      true, // skipped because authority gated
+					RpcMethod:      "UpdateParams",
+					Use:            "update-params-proposal [params]",
+					Short:          "Submit a proposal to update distribution module params. Note: the entire params must be provided.",
+					Example:        fmt.Sprintf(`%s tx distribution update-params-proposal '{ "community_tax": "20000", "base_proposer_reward": "0", "bonus_proposer_reward": "0", "withdraw_addr_enabled": true }'`, version.AppName),
+					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "params"}},
+					GovProposal:    true,
 				},
 				{
 					RpcMethod: "CommunityPoolSpend",
-					Skip:      true, // skipped because authority gated
+					Skip:      true, // skipped because deprecated in favor of protocolpool
```

Comment:
We should add this in because the protocolpool is optional now

Reply:
you will be rejected if it is enabled

Reply:
https://github.com/cosmos/cosmos-sdk/pull/24359/commits/d8e6537c8ad515ba33b2b40044974b1700ac1f8b